docs/_static/config_examples/sample-url-rewrite-annotation.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: extensions/v1beta1
```

Comment:
Is this the only url rewrite example? We should definitely have samples showing multi-path / multi-target examples as that's a complicated case.

Reply:
There are better examples in the solution docs I can pull in I was thinking of having simple examples here and then the more complicated examples in the solution docs.